app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationDialogFragment.kt

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.biometrics
+
+import android.app.Dialog
+import android.app.DialogFragment
+import android.arch.lifecycle.LifecycleObserver
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.support.annotation.RequiresApi
+import android.support.v4.content.ContextCompat
+import android.support.v7.app.AppCompatDialogFragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import org.mozilla.focus.R
+
+@RequiresApi(api = Build.VERSION_CODES.M)
+class BiometricAuthenticationDialogFragment : AppCompatDialogFragment(), LifecycleObserver {
+    private var mErrorTextView: TextView? = null
```

Comment:
the `m` naming convention is an old Java convention. We should rename this to `errorTextView` 😄

Reply:
You can access this through the fragment with `biometric_error_text` as well :)